core/src/error/report.rs

```diff
@@ -24,33 +25,41 @@ pub enum ErrorFormat {
 }
 
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
-pub struct ColorOpt(pub(crate) clap::ColorChoice);
+pub struct ColorOpt(pub(crate) colorchoice::ColorChoice);
```

Comment:
Do we even need this wrapper struct anymore? I think it was probably just there to avoid leaking `clap` into our public API, but maybe it's fine just to export `colorchoice::ColorChoice` since the crate is small and it's aligned exactly with what we want.

Reply:
Looks like the consequence is that the orphan rule strikes and we then need to convert the From impls to helper functions instead. Unless you see some nicer way out? 🤔 If not, I can revert or keep as is, whichever you prefer, and make some further tweaks if you like (e.g. if you want me to do some changes to the function names or whatever).

Reply:
I think either option is fine, let's not hold this PR up with bikeshedding.